src/pages/accounts/ForgotPassword.tsx

```diff
@@ -81,39 +121,182 @@ function ForgotPasswordPage() {
 		};
 	}, [email]);
 
-	const handleSubmit = async () => {
-		if (!validEmail(email)) {
-			setEmailErr(true);
+	const handleEmailStep = () => {
+		if (validEmail(email.value)) {
+			setEmail({ error: false });
+			setCodeAndPassStep(true); // Send validation code
+			Auth.forgotPassword(email.value)
+				.then((data) => console.log(data))
+				.catch((err) => console.log(err));
+		} else {
+			setEmail({ error: true, errorMessage: EMAIL_ERROR_MESSAGE });
 			return;
 		}
-		console.log(email);
+	};
+
+	const handleCodeAndPassStep = () => {
+		if (!validationCode.value) {
+			setValidationCode({
+				error: true,
+				errorMessage: VALIDATION_CODE_EMPTY_MESSAGE,
+			});
+		}
+		if (!password.value && !confirmPass.value) {
+			setPassword({ error: true, errorMessage: PASS_EMPTY_MESSAGE });
+		}
+		if (!validPass(password.value) && password.value !== '') {
+			setPassword({ error: true, errorMessage: PASS_ERROR_MESSAGE });
+			return;
+		}
+		if (
+			!(password.value === confirmPass.value) &&
+			confirmPass.value !== ''
```

Comment:
Is there a reason why it's `!(x === y)` on one half of the expression but `x !== y` on the other? === is still a bit of a grey concept for me so I'm just curious :]

Reply:
No, I don't think there is a difference in this case. I think when I was working it in my head the logic made sense this way, but I'll change it so its consistent haha